docs/conf.py

```diff
@@ -25,7 +25,7 @@
 # -*- coding: utf-8 -*-
 #
 # Procrustes documentation build configuration file, created by
-# sphinx-quickstart on Wed Apr 11 19:35:53 2018.
+# sphinx-quickstart on Wed Apr 11 19:35:53 2020.
 #
 # This file is execfile()d with the current directory set to its
```

Comment:
what is "execfile()d"? Looks weird.

Reply:
Thank you for pointing it out. `execfile()d` by Sphinx with the current directory set to its containing dir. More details can be found in the first few paragraphs, at https://pythonhosted.org/CodeChat/CodeChat/template/conf.py.html.